src/Console/Application.php

```diff
@@ -34,13 +40,60 @@ class Application extends BaseApplication
      */
     private $container;
 
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    /**
+     * @var bool
+     */
+    private $isXdebugLoaded;
+
+    /**
+     * @var bool
+     */
+    private $isDebuggerDisabled;
+
     public function __construct(Container $container, string $name = self::NAME, string $version = self::VERSION)
     {
         $this->container = $container;
+        $this->isDebuggerDisabled = empty((string) getenv(XdebugHandler::ENV_DISABLE_XDEBUG));
 
         parent::__construct($name, $version);
     }
 
+    public function run(InputInterface $input = null, OutputInterface $output = null)
+    {
+        if (null === $input) {
+            $input = new ArgvInput();
+        }
+
+        if (null === $output) {
+            $output = new ConsoleOutput();
+        }
+
+        $this->io = new SymfonyStyle($input, $output);
+
+        if (PHP_SAPI === 'phpdbg') {
+            $this->io->note(sprintf(self::RUNNING_WITH_NOTE, PHP_SAPI));
+        } elseif ($this->isXdebugLoaded) {
```

Comment:
I don't quite understand how it works. Just checked - it always `null` and is not used/set anywhere. Could you please explain?

Reply:
good catch! Shame on me I pushed not final version ^_^